Beyond08-SOW.md

```diff
@@ -0,0 +1,473 @@
+# Red Hat Beyond 08
+
+Statement of Work (SOW)
+***
+
+## Pre-requirements
+
+You will be working on a project during this course.
+
+The following tech stack will be used.
+
+If you don't have knowledge/experience in any of them, it is worthwhile to look
+them up. The more you know the easier it will be.
+
+* Linux OS - able to use basic command line tools such as cd, ls, cat, grep,
+  less, vi, ...
+* Python - if/else, for loops, functions, classes
+* Django - create a basic or sample project
+* Git and GitHub - create a repo, commit, push, pull requests (PR), merge
+* HTML/CSS (Bootstrap) - design a basic web page from scratch, including using
+  CSS classes for basic formatting
+* Testing- unit tests and functional tests with pytest
+* Virtualization - Spin up a VM using Vagrant and VirtualBox
+* Databases - Basic familiarity with database concepts
```

Comment:
Can add data modeling -  creating an ERD and DFD diagram

Reply:
@Omeramsc ^^

Reply:
I saw it as part of the demo instructions, and it should stay there, but it doesn't mean we cannot have it in the `Pre-requirements` as well, UML is just as important as the other stuff in this course.

Reply:
Ack